src/form/validated-form.tsx

```diff
@@ -36,18 +37,12 @@ export interface ValidatedFormProps {
  * @param ValidatedFormProps
  * @returns JSX.Element
  */
-export function ValidatedForm({ defaultValues, children, onSubmit, mode, ...rest }: ValidatedFormProps): JSX.Element {
+export function ValidatedForm({ defaultValues, form, children, onSubmit, mode, ...rest }: ValidatedFormProps): JSX.Element {
   const {
     handleSubmit,
     register,
-    reset,
-    setValue,
     formState: { errors, touchedFields, dirtyFields },
-  } = useForm({ mode: mode || 'onTouched', defaultValues });
-
-  useEffect(() => {
-    reset(defaultValues);
-  }, [reset, defaultValues]);
+  } = form;
```

Comment:
since the field is marked as optional now, shouldn't there be a default if not passed?

Reply:
NP.
What do you think about the approach?
Better before or after?